frontend/src/routes/index.js

```diff
@@ -33,6 +34,9 @@ function Routes() {
       <Route exact path={PATH.FAQ} element={<Faq />} />
       <Route exact path={PATH.RESEARCH} element={<Research />} />
       <Route exact path={PATH.CONTRIBUTORS} element={<Contributors />} />
+
+      <Route exact path={PATH[404]} element={<NotFound />} />
+      <Route path="*" element={<Navigate to={PATH[404]} />} />
```

Comment:
Oh sorry, I didn't mean to redirect to a 404 page. Just to show a 404 page for any unmatched path. I find it useful during development to have the location remain when it's invalid, so I can tell what's wrong about it.

Reply:
Got it. Fixed